docs/pages/preview/upcoming-releases.mdx

```diff
@@ -23,7 +23,8 @@ Application Access.
 Teleport 10.2 will bring file upload/download support to Desktop Access,
 host certificate support to Machine ID, Elasticsearch support to Database Access,
 automatic configuration of AD to Desktop Access, and remove the need for a PVC
```

Comment:
```suggestion
and automatic configuration of AD to Desktop Access. Teleport 10.2 will remove the need for a PVC
```
To break up this sentence a bit and preserve the parallel structure of the first list, which makes it slightly more readable.